content/docs/source-git/how-to-source-git.md

```diff
@@ -184,6 +185,10 @@ generating downstream patches, to be added in dist-git. As `.distro` is the
 place where the files related to packaging are stored, set the configuration
 to ignore this path, so that there are no patch files generated for it.
 
+`patch_generation_patch_id_digits` tells Packit how many digits should be used
+when adding `PatchN` tags to the spec-file. The patch in `acl` is in the form
+of `Patch1` (no leading zeros), so set this value to 0.
```

Comment:
Hmm, I would say that 1 in `Patch1` is one digit and would set the value to 1.
So now it's not clear to me whether it's "number of digits" or "number of leading zeros".

Reply:
Updated, please let me know if the new wording makes more sense.

This is a "minimum width in digits", where 0 means no minimum.